cdap-cli/src/main/java/co/cask/cdap/cli/CLIMain.java

```diff
@@ -248,10 +265,63 @@ public static void main(String[] args) {
         cliMain.tryAutoconnect();
         cliMain.updateCLIPrompt(cliConfig.getClientConfig());
 
-        if (commandArgs.length == 0) {
+        boolean ranScript = false;
+        boolean brokeFromScript = false;
+        if (launchOptions.getScriptFile() != null) {
+
+          ranScript = true;
+
+          ConnectionConfig connectionConfig = clientConfig.getConnectionConfig();
+          URI baseURI = connectionConfig.getURI();
+          URI uri = baseURI.resolve("/" + connectionConfig.getNamespace());
+          String cliPrompt = "\ncdap (" + uri + ")> ";
+
+          BufferedReader br = null;
+          try {
+            String line;
+            br = new BufferedReader(new FileReader(launchOptions.getScriptFile()));
+            // br = Files.newReader(launchOptions.getScriptFile(), Charset.UTF_8);
+            while ((line = br.readLine()) != null) {
+              String ln = line.trim();
+              String[] lnArr = ln.split("#");
```

Comment:
Added comment and semi-colon (multi-command) support for command line and script file command string interpretation.  Would like to add it to command prompt string interpretation, but could use some help in understanding the cli.startInteractiveMode() call and where the command prompt strings get interpreted.